src/Tronkell/Data/Parse.hs

```diff
@@ -9,13 +9,14 @@ import Tronkell.Game.Types as GT
 import Tronkell.Server.Types as ST
 import Data.Text as Text
 
-import Data.Aeson as A (ToJSON, object, (.=), toJSON, Value(..))
+import Data.Aeson as A
+import Data.Aeson.Types as AT
```

Comment:
Trivial comment, can be fixed separately -- if you're only ever using the import as `A.`..., you can use `import qualified`. An unqualified import of this kind brings the entire library namespace in as well as allows the qualified form of access.